### DISCUSSION TITLE
Thanks for being awesome, Patrick!

### DISCUSSION THREAD

**Discussion by nilspupils** [mcguirepr89/BirdNET-Pi#65] [General]
Seriously, look what you have achieved in a little more than one month. A nearly perfect system to easily use the best bird sound recognition system, now it is actually becoming citizen science. Making BirdNET useable for everyone is opening new worlds for bird lovers and scientists alike. 
Your will and enthusiasm to push forward in this project is absolutely inspiring, your endless offers of help, even for the most mundane of problems, very generous. 
So, can we, apart from thanking you, at least buy you a coffee?
Best, Nils

**Comment by mcguirepr89** [comment]
Thanks, Nils!! Those are incredibly kind words and knowing that my efforts are appreciated mean a lot to me. And I can say with no uncertainty that this project would not be what it has become without the contributions and feedback I get from users like you. For that reason, I would want us to _share_ a coffee!

And while I do LOVE coffee, and while it has played a huge behind-the-scenes role in BirdNET-Pi, I'm not able to accept any kind of compensation for my work on this project for fear of breaching the very new creative commons license that BirdNET and BirdNET-Lite are now under.

The time you have lent the project in testing and in contributing to discussions is all I can ask and hope for from users and my fellow birders :) WITH the additional request that once you start collecting some data, you share it with us!

Who knows, perhaps this note of gratitude will be seen by a future employer and you will have bought me lots of coffees :)

Either way, I hope someone see this and feels all the more welcome to test and share and ask questions so that BirdNET-Pi continues to improve.

My best regards as always, 
Patrick :swan:

**Comment by nilspupils** [comment]
I am really excited how this project is progressing. BirdNET-Pi will be very much welcomed not only by bird enthusiasts but also by the whole scientific community. Apart from being a lot of fun, BirdNET-Pi is a not only a serious reasearch tool, but also an easily accessible framework with many more applications. The BirdNET tensorflow model could be replaced with any other model to do whatever bioacoustic research you can imagine with your system.
When you think the time is right, you should really make this public, and especially let the scientific community know about it. Hit me up for some ideas about how to do this. 
Best, Nils

**Reply by mcguirepr89** [reply]
When it feels like the pre-installed image is as close to flawless as it can get, I will be ready and will take you up on the offer to help with ideas on how to let others know about BirdNET-Pi. Thank you for all of your support!!!!

**Comment by mcguirepr89** [comment]
@nilspupils @CaiusX Looking back at this makes me smile :) Thanks again!!!!!!!!!!!!

**Comment by mcguirepr89** [comment]
Thank you, @nilspupils and @CaiusX for being my first sponsors!!!! I APPRECIATE YOU SO MUCH!

**Comment by oliverk71** [comment]
I have to admit, I am overwhelmed how this project develops! :) @mcguirepr89 
The new birdnetpi is just great! I like it so much that I thought about a T-shirt with the birdnetpi logo :)

**Reply by mcguirepr89** [reply]
Thank you so much!!! The T-Shirt would be super awesome, BUT I don't own the logo, so you'd have to make sure the back of the shirt had some links for attribution :)

https://www.flaticon.com/premium-icon/red-cardinal_3498899?term=red%20cardinal&page=1&position=1&page=1&position=1&related_id=3498899&origin=search

**Reply by oliverk71** [reply]
According to the website, it should be something like "This cover has been designed using resources from Flaticon.com". Maybe I can shorten this a little bit. 

But, since this is a free logo, which everybody can use, it is not limited birdnetpi: How about creating a unique logo? Maybe one in our community can do it? I can try, but I am not good at creating logos. :D

**Reply by mcguirepr89** [reply]
That would be cool! I'm crap at design, but would love for the logo to still represent a northern cardinal, as that is the state bird for my home state of North Carolina as well as my current state of residence, Virginia :)
